### PR TITLE
Fix set plugin setting

### PR DESCRIPTION
**Pull request by Alexander-Dubrawski**
**Does your pull request solve a problem? Please describe:**  
This Pr solves the bug that the frontend sends the display name instead of the setting name to update the plugin settings. I adjusted the backend that it returned the `setting name` as well as the `display name`. This Pr also adjusted the frontend to access the `display name` attribute instead of the `setting name` attribute. 

## Todo:

- [x] When approved run `pre-commit run --all-files`

### CODE REVIEW

**Comment by Bensk1** [comment]
> [ ] When approved run `pre-commit run --all-files`

Is this standard procedure for PRs now? Are they failing until you run this command?

File: hyrisecockpit/api/app/plugin/schema.py [hunk 1]
```diff
@@ -11,9 +11,7 @@ class PluginSchema(Schema):
     """Schema of a Plugin."""
 
     name = String(
-        description="Identifier of the Plugin.",
-        required=True,
-        default="CompressionPlugin",
+        description="Identifier of the Plugin.", required=True, default="Compression",
```
Comment (Bensk1) [inline hyrisecockpit/api/app/plugin/schema.py]: Although it was already `CompressionPlugin` before your change, such a specific default seems odd.

File: hyrisecockpit/api/app/plugin/schema.py [hunk 2]
```diff
@@ -46,6 +44,11 @@ class PluginSettingSchema(PluginSettingBaseSchema):
         required=True,
         default="This text describes a setting.",
     )
+    display_name = String(
+        description="Name of the setting that shall be displayed.",
+        required=True,
+        default="Memory Budget (MB)",
```
Comment (Bensk1) [inline hyrisecockpit/api/app/plugin/schema.py:52]: see above.

Comment (Bensk1) [reply hyrisecockpit/api/app/plugin/schema.py:52]: Maybe, I don't understand `default` correctly.

**Comment by Alexander-Dubrawski** [comment]
> > [ ] When approved run `pre-commit run --all-files`
> 
> Is this standard procedure for PRs now? Are they failing until you run this command?

No, it's not standard. Something just went wrong in the last merge with the `dev`. The hooks in the dev were updated, but for one hook the update is missing.

File: hyrisecockpit/api/app/plugin/schema.py (hunk 1 above)
Comment (Alexander-Dubrawski) [reply hyrisecockpit/api/app/plugin/schema.py]: The default is used in `swagger` to easily test the endpoint. 

Screenshot 2020-09-25 at 16 35 28

File: hyrisecockpit/api/app/plugin/schema.py (hunk 2 above)
Comment (Alexander-Dubrawski) [reply hyrisecockpit/api/app/plugin/schema.py:52]: The default here will be used in `swagger` as an example.

Screenshot 2020-09-25 at 16 39 10

**Comment by Bensk1** [comment]
Thanks for clearing that up.

File: hyrisecockpit/api/app/plugin/schema.py (hunk 2 above)
Comment (Bensk1) [reply hyrisecockpit/api/app/plugin/schema.py:52]: Okay, never mind.

**Review by Bensk1 — APPROVED** [review]

**Pull request merged**